flowers.json

```diff
@@ -500,7 +500,7 @@
       ],
       "blooming_period": "--07-01/--09-01",
       "description": {
-        "summary": "Veronicastrum virginicum, or Culver’s root, is a perennial flower with tall stalks and small white flowers arranged in spikes. Individual flowers bloom for over a month, attracting bees, wasps, and soldier beetles. Culver’s root blooms from July to September and grows in woodland edge or prairie habitats.",
+        "summary": "Culver’s root is a perennial flower with tall stalks and small white flowers arranged in spikes. Individual flowers bloom for over a month, attracting bees, wasps, and soldier beetles. Culver’s root blooms from July to September and grows in woodland edge or prairie habitats.",
         "bees_attracted": "Culver’s root commonly attracts these bees: Lasioglossum (small sweat bees), Hylaeus (yellow-faced bees), Halictus (sweat bees), Megachile (leafcutter bees), Melissodes (long-horned bees), and Bombus (bumble bees)."
```

Comment:
Looks like we are using the "Typesetter's apostrophe" here (`’`) (U+2019) but we are using the regular "Typewriter apostrophe" in the actual name (`'`) (U+0027). We should decide to use one of the two and make sure that change is applied everywhere. This probably happened due to copying from a word processor.